diff --git a/buildscript/BuildToolChain b/buildscript/BuildToolChain
index 6dcca35..e949f9d 100755
--- a/buildscript/BuildToolChain
+++ b/buildscript/BuildToolChain
@@ -1084,7 +1084,7 @@ build_uboot()
 	# do the cd to avoid pwd warnings with `make`
 	ver=$(cd /; printf 'e:;@echo $(U_BOOT_VERSION)\ninclude '$DIR_UBOOT'/Makefile\n' | ${MAKE} -f -)
 
-	echo_date "U-Boot: building helper tools (${ver})"
+	echo_date "U-Boot: building helper tools"
 	change_dir "$DIR_UBOOT"
 	DIR_UBOOT_BUILD="$DIR_BUILD/u-boot_build"
 
@@ -1369,7 +1369,7 @@ build_uclibc_headers()
 
 	ver=$(printf 'e:;@echo $(VERSION)\ninclude '$DIR_UCLIBC_SOURCE'/Rules.mak\n' | ${MAKE} -f - 2>/dev/null)
 
-	echo_date "${target}: uClibc: installing headers (${ver})"
+	echo_date "${target}: uClibc: installing headers"
 
 	build_uclibc_config "${target}" "${sysroot}" .config-used-for_${target}-headers
 
